cmd/explored/main.go

```diff
@@ -61,7 +61,7 @@ var cfg = config.Config{
 		BatchSize: 1000,
 	},
 	Log: config.Log{
-		Level: "info",
+		Level: "debug",
```

Comment:
is this a debug leftover or do we really want to change the default log level to `debug`, we normally have it at `info`